### DISCUSSION TITLE
How to skip exceptions and get results?

### DISCUSSION THREAD

**Discussion by asanovr** [nette/latte#365] [Q&A]
Hi, I sometimes have a template that can not have some variables. In my current implementation, I can't do it but I see that the fiddle already does it, for [example](https://fiddle.nette.org/latte/#4b9bbcc0f4). How can I handle it?

**Comment by eydun** [comment]
You can do like this:
```latte
{ifset $aaa}
{foreach $aaa as $v}
	xxxxx
{/foreach}
{/ifset}
{$test ?? ''}
```

**Comment by dg** [comment]
Try 
```latte
{foreach $aaa ?? [] as $v}
```

**Reply by asanovr** [reply]
Thanks! But I think fiddle handles errors in another way and checking every variable is not suited to me. 

I will try to explain:
I've templates and questions created by users. Users write templates using these questions(each question has an alias property that is used as a variable). Sometimes users make a mistake with a variable name or a variable can be deleted from a system but not deleted from a template. I'm trying to figure out how to handle this and render templates, even if a template contains undefined variables.
